client/examples/publish_bincode.rs

```diff
@@ -4,7 +4,7 @@ use selium::prelude::*;
 use selium::std::codecs::BincodeCodec;
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
What's this `Clone` for?

Reply:
This was a mistake made in the generic constraints for the `KeepAlive` Sink impl 🤦‍♂️ 

Fixed!